crates/core/src/host/wasm_common/module_host_actor.rs

```diff
@@ -439,6 +442,7 @@ impl<T: WasmInstance> WasmModuleInstance<T> {
         let ctx = ExecutionContext::reducer(address, ReducerContext::from(op.clone()));
         // run the call_reducer call in rayon. it's important that we don't acquire a lock inside a rayon task,
         // as that can lead to deadlock.
+        // TODO: What to do with this `ctx`?
         let (ctx, mut tx, result) = rayon::scope(|_| tx_slot.set(ctx, tx, || self.instance.call_reducer(op, budget)));
```

Comment:
I need to know how deal with this `ctx`

Reply:
We should just remove it since it's already embedded inside of `tx`.